src/patternfly/components/FormControls/examples/form-controls-input-example.hbs

```diff
@@ -0,0 +1,8 @@
+{{#> form-controls controlType="input" input="true" pf-c-form-control--modifier="pf-m-disabled" type="text" id="textInput4" disabled="yes" placeholder="Disabled input example"}}
+{{/form-controls}}
+{{#> form-controls controlType="input" input="true" type="text" id="textInput1" placeholder="Standard input example" }}
+{{/form-controls}}
+{{#> form-controls controlType="input" input="true" type="text" id="textInput3" required="yes" placeholder="Required input example" }}
+{{/form-controls}}
+{{#> form-controls controlType="input" input="true" pf-c-form-control--modifier="pf-m-login" type="text" id="textInput4"  placeholder="Login input example"}}
+{{/form-controls}}
```

Comment:
Nitpicky, but should 'yes' values be 'true' instead?

Reply:
👍